### ISSUE TITLE
Verify OS, Configure OS with PI and Navio 

### ISSUE THREAD

**Issue by ConleyGao** [ToddHuaJi/FlappyDrone#42]
very, set configuration
Details can be found at: https://docs.google.com/document/d/1lmKDjlOBGNS6KsVinD3MQjvMlFBXX2Ow8JaHm9-rOOk/edit?usp=sharing

**Comment by ConleyGao** [comment]
1. setup the keyboard config to proper us Key board 
2. updated OS

**Comment by ConleyGao** [comment]
3. updated Navio

**Comment by ConleyGao** [comment]
pi itself is fine , Someone else do some configuration with the Ardupilot ex GCS IP,  bluetooth

**Comment by ToddHuaJi** [comment]
> pi itself is fine , Someone else do some configuration with the Ardupilot ex GCS IP, bluetooth

which step is it on the installation guide?

**Comment by ToddHuaJi** [comment]
We need a simple router, in this case a cellphone hotspot would be perfert to connect.

*Second attempt on connecting with Mission Planner(MP)** 
Connecting the pi with ethernet cable failed
I used my cell phone as hotspot and connected my laptop and the pi to my hotspot network. I did change the ip address in the process. This process takes a couple of minutes and it is irritating, but it works

**Exploring the possibility of switching wifi**
Since we don’t have any GUI, the best I can do now is the following:

1. Turn on my hotspot
2. reboot/turn on the pi
3. The pi should automatically connect to the hotspot 
4. Check for IP address
5. Start connection

To switch back to eduroam:

- Turn of hotspot
- Reboot
